src/helpers.ts

```diff
@@ -35,16 +42,24 @@ export function getYamlFromJson(json: string): string {
 
 export function getJsonFromYaml(yaml: string): string {
 	const schema = getConfig<Configs['YamlSchema']>(ConfigId.YamlSchema);
+	const nConvention = getConfig<Configs['EnforceNamingConvention']>(ConfigId.EnforceNamingConventionJson);
```

Comment:
I'm not sure what "n" refers to here? Why did you decide to name this `nConvention`?

Reply:
'naming' I'll rename it to just convention!